app/src/main/java/com/github/libretube/ui/activities/MainActivity.kt

```diff
@@ -280,6 +280,22 @@ class MainActivity : BaseActivity() {
         return super.onPrepareOptionsMenu(menu)
     }
 
+    private fun isSearchInProgress(): Boolean {
+        if (!::navController.isInitialized) return false
+        val id = navController.currentDestination?.id ?: return false
+        return ((id == R.id.searchFragment) || (id == R.id.searchResultFragment))
```

Comment:
I'd prefer something like
```suggestion
        return  id in listOf(R.id.searchFragment, R.id.searchResultFragment)
```
for better readability.

Reply:
Thanks, will update same commit with your suggestion.